js/js.go

```diff
@@ -1,28 +1,60 @@
-//  | --------------------- | --------------------- | ------------------------------- |
-//  | bool                  | Boolean               | bool                            |
-//  | integers and floats   | Number                | float64                         |
-//  | string                | String                | string                          |
-//  | []int8                | Int8Array             | []int8                          |
-//  | []int16               | Int16Array            | []int16                         |
-//  | []int32, []int        | Int32Array            | []int                           |
-//  | []uint8               | Uint8Array            | []uint8                         |
-//  | []uint16              | Uint16Array           | []uint16                        |
-//  | []uint32, []uint      | Uint32Array           | []uint                          |
-//  | []float32             | Float32Array          | []float32                       |
-//  | []float64             | Float64Array          | []float64                       |
-//  | all other slices      | Array                 | []interface{}                   |
-//  | arrays                | see slice type        | see slice type                  |
-//  | functions             | Function              | func(...interface{}) *js.Object |
-//  | time.Time             | Date                  | time.Time                       |
-//  | -                     | instanceof Node       | *js.Object                      |
-//  | maps, structs         | instanceof Object     | map[string]interface{}          |
+// Internalization
 //
-// Additionally, for a struct containing a *js.Object field, only the content of the field will be passed to JavaScript and vice versa.
+// When values pass from Javascript to Go, a process known as internalization,
+// the following conversion table is applied:
+//
+//  |----------------+---------------+-------------------------+--------|
+//  | Go target type | Translation   | Javascript source value | Result |
+//  |----------------+---------------+-------------------------+--------|
+//  | string         | UTF16 -> UTF8 | null                    | ""     |
+//  |                |               | undefined               | ""     |
+//  |                |               | ""                      | ""     |
+//  |                |               | new String("")          | ""     |
+//  |                |               | "ok" †                  | "ok"   |
+//  |                |               | new String("ok") †      | "ok"   |
+//  |----------------+---------------+-------------------------+--------|
+//  | bool           | none          | null                    | false  |
+//  |                |               | undefined               | false  |
+//  |                |               | false †                 | false  |
+//  |                |               | new Boolean(false) †    | false  |
```

Comment:
Is it worth including `true` examples here, for the sake of completeness, similar to the non-zero value examples for strings?